composer.json

```diff
@@ -5,6 +5,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.1",
+        "setono/doctrine-orm-batcher-bundle": "^0.2.0",
```

Comment:
When do you expect to have a stable release? I'm not a fan of adding 'non-stable' dependencies.